core/src/main/resources/schema/infinispan-config-15.0.xsd

```diff
@@ -809,7 +809,7 @@
         </xs:documentation>
       </xs:annotation>
     </xs:attribute>
-    <xs:attribute name="names-as-tags" type="xs:boolean" default="${GlobalMetrics.namesAsTags}">
```

Comment:
IIRC we have different other cases of camel-cased default

Reply:
I must admit it is also a bit confusing that the variable name is all upper case with underscores and the replaced attribute name is all lower case with dashes.

Reply:
probably, but it is derived from the attribute name which is an enum which is converted from snake-uppercase to kebab-lowercase